charts/vald-helm-operator/README.md

```diff
@@ -69,7 +69,7 @@ Configuration
 | rbac.name | string | `"vald-helm-operator"` | name of roles and rolebindings |
 | reconcilePeriod | string | `"1m"` | reconcile duration of operator |
 | replicas | int | `2` | number of replicas |
-| resources | object | `{}` | k8s resources of pod |
+| resources | object | `{}` | kubernetes resources of pod |
```

Comment:
If you'd like to revise here, it is recommended to review values.yaml in helm operator.
This markdown is automatically generated from it.

Reply:
Thank you your feedback.
I fixed comment of values.yaml.